File: Models/Education.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MtpApp.Models
{
    public class Education
    {
        public int Id { get; set; }

        [Required]
        public int ClientId { get; set; }

        [Required]
        [StringLength(255)]
        public string Level { get; set; }

        [StringLength(255)]
        public string Grade { get; set; }

        [StringLength(255)]
        public string Subject { get; set; }

        [StringLength(255)]
        public string Year { get; set; }

        [StringLength(255)]
        public string SchoolName { get; set; }
        public string Description { get; set; }
    }
}

File: spring-backend/src/main/java/com/mtp/api/models/Education.java
package com.mtp.api.models;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Entity
@Table(name = "Educations")
@Data
public class Education {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "Id")
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ClientId", insertable = false, updatable = false)
    private Client client;

    @NotNull
    @Column(name = "ClientId")
    private Integer clientId;

    @Size(max = 255)
    @Column(name = "Level")
    private String level;

    @Size(max = 255)
    @Column(name = "Grade")
    private String grade;

    @Size(max = 255)
    @Column(name = "Subject")
    private String subject;

    @Size(max = 255)
    @Column(name = "Year")
    private String year;

    @Size(max = 255)
    @Column(name = "SchoolName")
    private String schoolName;

    @Column(name = "Description")
    private String description;
}
